src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/binary_convolution_gpu_1x1.cl

```diff
@@ -22,6 +22,15 @@ KERNEL(binary_convolution_1x1)(const __global INPUT0_TYPE* input,
 #endif
                                uint split_idx)
 {
+//This is workaround code for compiler bug in unit test "binary_convolution.basic_convolution_1x1_single_packed_channel". Delete it when the bug fixed.
+#if DG2
+    volatile uint _FILTER_MASK = FILTER_MASK;
+    #undef FILTER_MASK
+    volatile uint FILTER_MASK = _FILTER_MASK;
```

Comment:
This WA (and especially the magic with mask) needs some comment with explanation...

Reply:
Removed volatile FILTER_MASK because giving volatile to input is enough to pass test.